models/evms/evms_traces.sql

```diff
@@ -17,6 +18,7 @@
      , ('fantom', source('fantom', 'traces'))
      , ('optimism', source('optimism', 'traces'))
      , ('arbitrum', source('arbitrum', 'traces'))
+     , ('celo', source('celo', 'traces'))
 ] %}
 
 SELECT *
```

Comment:
Beware that celo.traces contains the additional tx_from and tx_to columns, so this union must be made with explicit column names. It's a good practice to write out all the column names explicitly, as then the spell won't break if additional columns are introduced in some of the upstream table.

Reply:
Good catch, I was too lazy to define all the columns in each model when creating these tables, just defined them in all tables